README.md

```diff
@@ -36,7 +36,14 @@ TODO: npm での使用方法を書く
 
 NES.css only provides components. You will need to define your own layout.
 
-The default font is Press Start 2P which only supports English characters. When you're using this framework with any language except English, please use another font.
+The default font is [Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P?selection.family=Press+Start+2P) which only supports English characters. When you're using this framework with any language except English, please use another font. Please follow the [instructions](https://developers.google.com/fonts/docs/getting_started) from Google Fonts about how to include them, or simply include it as below:
+
+```html
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
+    <link href="https://unpkg.com/nes.css/css/nes.css" rel="stylesheet" />
```

Comment:
please add below code. 🙏 

```suggestion
    <link href="https://unpkg.com/nes.css/css/nes.css" rel="stylesheet" />
    <style>
        html, body, pre, code, kbd, samp {
            font-family: "font-family you want to use";
        }
    </style>    
```

this code overwrite `font-family` in `reboot.scss`.
https://github.com/BcRikko/NES.css/blob/ded4ca4aa9a1c1ee217e9abc9ca3102d2c49fcc5/scss/base/generic.scss#L3-L10

Reply:
added in [928601fc5ac](https://github.com/BcRikko/NES.css/pull/48/commits/928601fc5acf19f7d707e018e679b06f1bd95657) 🙏